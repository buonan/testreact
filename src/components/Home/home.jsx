import React from 'react'

const Home = props => <div>{props.name}</div>;

export default Home
