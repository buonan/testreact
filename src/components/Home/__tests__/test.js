import React from 'react'
import { mount, shallow, configure } from 'enzyme'
import Adapter from 'enzyme-adapter-react-16';
import Home from '../home'

configure({adapter: new Adapter()});
describe('Home tests', ()=> {
  it('props test', ()=>{
    const component = mount(<Home name='Chris' />)
    expect(component.find('Home').exists()).toBe(true)
    expect(component.props().name).toBe('Chris')
  })

  it('should match snapshot and styles for default props', ()=>{
    const component = mount(<Home name='Bob' />)
    console.log(component)
    expect(component).toMatchSnapshot()
  })
})