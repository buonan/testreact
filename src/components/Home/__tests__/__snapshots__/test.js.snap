// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home tests should match snapshot and styles for default props 1`] = `
<Home
  name="Bob"
>
  <div>
    Bob
  </div>
</Home>
`;
